test/e2e/certificates_test.go

```diff
@@ -435,6 +452,202 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("dns-01 challenge using explicit credentials on ibm cloud CIS", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
```

Comment:
It's better to move this case into `Context("dns-01 challenge using explicit credentials", func() { ...`

https://github.com/openshift/cert-manager-operator/blob/6833a9e979e0601713f724800be44f7b4a8792d6/test/e2e/certificates_test.go#L65

And rename the `It` description.

As below:

```
Context("dns-01 challenge using explicit credentials", func() {
    It("should obtain a valid LetsEncrypt certificate", func() {
        ...
    }
+
+   It("should obtain a valid LetsEncrypt certificate in IBM Cloud CIS", func() {
+       ...
+   }
}
```

Reply:
We will update the code as such, thank you